Challenge-1/src/constants.ts

```diff
@@ -0,0 +1,14 @@
+export const CREATE_BOT_FUNCTION =
+  "function createAgent(uint256 agentId, address, string metadata, uint256[] chainIds)";
+
+export const UPDATE_BOT_FUNCTION =
+  "function updateAgent(uint256 agentId, string metadata, uint256[] chainIds)";
+
+export const BOT_UPDATE_EVENT =
+  "function _agentUpdate(uint256 agentId, string newMetadata, uint256[] newChainIds)";
```

Comment:
This is not used, so it should be removed (Should also be removed from the `provideHandleTransaction`).